static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -35,6 +39,29 @@ define([
                     bean.on(el, 'click', askQuestion);
                 });
             }
+
+            var storyQuestionsComponent = document.querySelector('.js-view-tracking-component');
+    
+            mediator.on('window:throttledScroll', function onScroll() {
+                var height = detect.getViewport().height;
+                var coords = storyQuestionsComponent.getBoundingClientRect();
+                var isStoryQuestionsInView = coords.top + window.scrollY > 0 && coords.bottom < height;
+                
+                if( isStoryQuestionsInView ) {
+                    var atomId = $('.js-storyquestion-atom-id').attr('id');
+
+                    if (atomId) {
+                        ophan.record({
+                            atomId: atomId,
+                            component: '',
```

Comment:
We only really care about the atom id in terms of the roll up in Ophan for analytics.